package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-01-node-ecosystem",
+  "version": "1.0.0",
+  "description": "![CF](https://camo.githubusercontent.com/70edab54bba80edb7493cad3135e9606781cbb6b/687474703a2f2f692e696d6775722e636f6d2f377635415363382e706e67) 401 JS -- Class 01 node ecosystem  ===",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
```

Comment:
This line should be `"test": "mocha"`. `mocha` is the command that runs your tests.